routes/team/teamSecure.js

```diff
@@ -2,47 +2,45 @@ const express = require('express');
 const router = express.Router();
 const { TeamInfo, UserInfo } = require('../../model/model');
 require('dotenv').config(); //initialize dotenv
-let ObjectId = require("bson-objectid");
-const { validateNonNullStringHashID, validateNonNullNumberID } = require('../auth/validation');
+const { validateNonNullNumberID, validateEmail } = require('../auth/validation');
 
 // Create new team
 router.post('/', async (req, res) => {
-    /*
-    - Max / min amount of team members??
+    /* 
+        Questions:
+            - Max / min amount of team members
+            - Shouldn't we ensure all the users are from the same university?
+            - Shouldn't this be sending out invites rather than auto-adding users?
+            - How do we ensure that the user creating the team doesn't put in a different university's ID?

```

Comment:
okay so 1. I don't remember but i think it was like max of 5? idk it might be in the docs somewhere
2. yes i think so 
3. yes? seems like thats the right idea 
4. maybe we handle it on the frontend like have it autofill w their uni id and not let them change it? unsure of how the frontend will look but i think we can figure it out